package.json

```diff
@@ -1,32 +1,45 @@
 {
   "name": "serverless-log-forwarding",
   "version": "2.0.0",
+  "engines": {
+    "node": ">=12"
+  },
   "description": "a serverless plugin to forward logs to given lambda function",
-  "main": "index.js",
+  "main": "dist/src/index.js",
   "directories": {
     "test": "test"
   },
   "dependencies": {
-    "underscore": "^1.8.3"
+    "@types/underscore": "^1.11.4",
+    "underscore": "^1.11.4"
   },
   "devDependencies": {
+    "@types/chai": "^3.5.0",
+    "@types/mocha": "^8.2.0",
+    "@types/shelljs": "^0.8.11",
+    "@types/randomstring": "^1.1.8",
+    "@types/async-retry": "^1.3.3",
+    "async-retry": "^1.3.3",
     "aws-sdk": "^2.1061.0",
     "chai": "^3.5.0",
-    "eslint": "^8.7.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-react": "^7.28.0",
-    "istanbul": "^0.4.5",
     "mocha": "^9.1.4",
+    "nyc": "^15.1.0",
     "randomstring": "^1.2.2",
-    "serverless": "^3.1.0",
-    "shelljs": "^0.8.5"
+    "serverless": "<4",
+    "shelljs": "^0.8.5",
+    "ts-node": "^10.4.0",
+    "typescript": "^4.5.5",
+    "eslint": "^8.7.0",
+    "@typescript-eslint/parser": "^5.11.0",
+    "@typescript-eslint/eslint-plugin": "^5.11.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-typescript": "^16.1.0"
```

Comment:
You can do alphabetical ordering of `devDependencies` it will be better to read

Reply:
addressed